static/js/components/NewTokenForm.jsx

```diff
@@ -24,14 +24,14 @@ import * as ProfileActions from '../ducks/profile';
 // const NewTokenForm = ({ acls, groups }) => {
 const NewTokenForm = ({ acls }) => {
   const dispatch = useDispatch();
-  const defaultValues = {
-    name: '',
-    acls: Array(50).fill(false) // This should be `acls.length`, but see https://github.com/react-hook-form/react-hook-form/issues/1558
-  };
 
-  const { handleSubmit, register, errors, reset, control } = useForm({
-    defaultValues
-  });
+  const { handleSubmit, register, errors, reset, control } = useForm();
+
+  useEffect(() => {
+    reset({
+      acls: Array(acls.length).fill(false)
+    });
+  }, [reset, acls]);
```

Comment:
Why is `reset` included here? Isn't that a static function? Does this work as expected without `reset` here?

Reply:
Can confirm that just `[acls]` works as expected, and I think is best here, as this hook isn't watching the value of `reset`, just `acls`.

Reply:
```suggestion
  }, [acls]);
```